dataservice_publisher/app.py

```diff
@@ -14,6 +14,13 @@
 from .resources.ping import Ping
 from .resources.ready import Ready
 
+DEFAULT_CONTENT_TYPE = {"text": "text/turtle", "application": "application/ld+json"}
```

Comment:
Er ikke `application/rdf+xml` en mer normal default for application?

Reply:
Man trenger strengt tatt også ikke gjenta seg selv, slik at dette mappet kan være fra _MIME_type_ til _MIME_subtype_. Ikke noen stor fordel utover at man unngår potensielle skrivefeil.

Reply:
Har endra til `application/rdf+xml` i 7ae6162424310b023545b445cb0790be3435659b